### PR TITLE
Home task: HTML 01 HTML 02

### PR DESCRIPTION
**Pull request by MariaYakubovich**
Could You please check my home work?

Thank You in advance

### CODE REVIEW

File: mYakubovich/HTML_02/Python_01_02.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+		состав большинства дистрибутивов Linux, на него часто не обращают внимания при выборе языка
+		для изучения. В этой статье мы рассмотрим создание консольных приложений, а в следующей —
+		программирование графических интерфейсов (<u>GUI</u>). Давайте создадим простую программу прямо
+		сейчас.</p>
+		<H2 id="programm">Наша первая программа</H2>
+		<p><A HREF="#contents">К содержанию</A></p>
+		<p>Наберём код в текстовом редакторе (например, в <strong>gedit</strong>). Потом мы разберёмся, что делает каждая
+		строка и пойдём дальше.</p>
+		<p>Введите эти 4 строки:</p>
+		<p><code>
```
Comment (pluhin) [inline mYakubovich/HTML_02/Python_01_02.html:32]: Do not use tag p for non-human content

File: mYakubovich/HTML_02/Python_01_02.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+			#!/usr/bin/env python<br>
+			print 'Hello. I am a python program.'<br>
+			name = raw_input("What is your name?")<br>
+			print "Hello there, " + name + "!"</code></p>
+		<p>Это всё. Сохраните файл как <strong>hello.py</strong> в любом месте. Я предлагаю сохранить его в папке
+		<b>python_examples</b> в домашнем каталоге. Этот пример показывает, как просто написать программу на
+		Python. Для запуска программы нужно сделать её файл исполняемым. Для этого введите в терминале</p>
+<pre><b><u><em>chmod +x hello.py</em></u></b></pre>
+		<p>в папке, в которой вы сохранили программу. Теперь запустите её.</p>
+		<p><code>
```
Comment (pluhin) [inline mYakubovich/HTML_02/Python_01_02.html:42]: Do not use tag p for non-human content

File: mYakubovich/HTML_02/Python_02_02.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+		программ.</p>
+		<H2 id="list">Списки</H2>
+		<p><A HREF="#contents">К содержанию</A></p>
+		<p>Давайте рассмотрим другой тип переменных, который называется <i>списки</i>. В других 
+		языках программирования они называются <i>массивами</i>. Если мы обратимся к аналогии с
+		коробками для обуви, то массив (список) представляет собой несколько склеенных друг с
+		другом коробок с одинаковыми вещами. Например, в одной коробке лежат вилки, в 
+		другой — ножи, в третьей — ложки. Посмотрим на простой список— список месяцев. Мы
+		запрограммируем его так:</p>
+<pre><br>
```
Comment (pluhin) [inline mYakubovich/HTML_02/Python_02_02.html:33]: in pre is it not necessary to use br, just add new line

**Review by pluhin — CHANGES_REQUESTED** [review]

**Comment by MariaYakubovich** [comment]
I wanted to add my corrections at hw. should i make them? or i may continue with the next tasks?
Thank you in advance

**Pull request merged**